varya/assets/sass/components/header/_config.scss

```diff
@@ -4,11 +4,14 @@
 	--branding--color-link-hover: var(--global--color-primary-hover);
 	--branding--title--font-family: var(--global--font-primary);
 	--branding--title--font-size: var(--heading--font-size-h1);
-	--branding--title--font-weight: normal;
+	--branding--title--font-weight: 700;
 	--branding--description--font-family: var(--global--font-secondary);
 	--branding--description--font-size: var(--global--font-size-sm);
 	--branding--description--font-family: var(--global--font-secondary);
 
+	--branding--logo--border-radius: 50%;
+	--branding--logo--max-width: 128px;
```

Comment:
I think these variable additions make sense.

Reply:
Just noting that when we use the `--branding--logo--max-width: 128px;` variable, it needs to match the what appears in functions.php here: https://github.com/Automattic/themes-workspace/blob/update/site-branding/varya/functions.php#L88-L89. And its probably worth adding `--branding--logo--max-height` as well. 

I’m a little unsure about this variable `--branding--logo--border-radius: 50%;` and having the site-logo cropped by default. This probably deserves a deeper discussion/solution, but users have tons of different kinds of logos and if someone uploads a type-driven logo, they won’t understand why it gets cropped into a circle and also won’t have a way to directly edit it. To keep that from happening it may be better to remove the CSS cropping and create a pre-cropped PNG for the logo instead. That way users can pretty much upload anything and it’ll just display it as-is. 

We’ll have to deal with this differently with FSE, but for the customizer option, I think we’ll be better off leaning toward more flexibility.

Reply:
> I’m a little unsure about this variable --branding--logo--border-radius: 50%; and having the site-logo cropped by default. This probably deserves a deeper discussion/solution, but users have tons of different kinds of logos and if someone uploads a type-driven logo, they won’t understand why it gets cropped into a circle and also won’t have a way to directly edit it. To keep that from happening it may be better to remove the CSS cropping and create a pre-cropped PNG for the logo for the theme instead. That way users can pretty much upload anything and it’ll just display it as-is.

I'm not _totally_ sure about this one. It's a design choice by the theme, and I think it's okay to leave it in. I've removed it for now just to get this through quickly though.

Reply:
Wait, I think I misspoke in the previous comment :-). I like the circular logo as a design choice—it should definitely stay circular for the demos and theme screenshots in this theme. 

I’m only suggesting we implement `site-logo` in a different way that offers users more visual variety so they aren’t _forced_ to have circular logo if they don’t want one. Users don’t always understand the dynamics of the logo graphics they have. They’ll take their logo as-is, upload it to the customizer and when ([this happens](https://user-images.githubusercontent.com/709581/77959986-854fef00-72a5-11ea-9936-fa950fc8956d.png)) they won’t understand why or how to fix it—it just looks broken to them. If we remove the circular crop, _any_ image can be a logo, which is what users expect (even though it often comes out looking bad). 

Another thing to consider is that `--branding--logo--border-radius: 50%` suggests that a child theme might want to change the 50% to something else like 4px, or 33%. But when it comes to displaying the logo, I can only imagine 50% or 0 being useful values in the context of Varya. If only two values would ever be used there, and those values aren’t tied to nay other, similar design decisions, then it doesn’t necessarily _need_ to be a part of the system. It can just be normal CSS and be just as effective in a child theme or the parent theme.